whitepapers/business-value/Introduction.md

```diff
@@ -0,0 +1,119 @@
+
+- Freedom to run the program as you wish, for any purpose
+- The freedom to study how the program works, and change it to make it do what you wish
+- The freedom to redistribute copies so you can help others
+- The freedom to distribute copies of your modified versions to others
+
+The open source software movement takes the philosophy of the free software movement and codifies it, creating a legal framework around free software to make it more palatable to businesses – particularly their legal departments. 
+
+The organization in charge of translating the philosophy of open source software into concrete legal terms is the [Open Source Initiative](https://opensource.org), also known as OSI. All software is released under a software license, and the OSI determines which licenses are open source licenses. You can see the list of OSI-approved licences [here](https://opensource.org/licenses). 
+
+Open source software is simply software that is released under an OSI-approved license. 
+
+It can be published anywhere, it can be created and maintained by volunteers or employees at AWS, it can be used for good and it can be used for evil. It can be produced by a single individual or a group, it can be a product of collaboration between multiple companies or it can be created entirely by a single company. Open source software is not fundamentally ethically or morally superior to proprietary software; it is a development model that can have both advantages and disadvantages. 
+
+If the software is published under an OSI-approved license, it is open source software. Conversely, software that is not published under an OSI-approved license is not open source. 
+
+The above point is important, because there are actors in the ecosystem that have tried to muddy the definition of open source, or have tried to position open source as a spectrum. Open source is not a spectrum; either a piece of software has an OSI-approved license or it does not. 
+
+You can see the full description of the [criteria OSI uses](https://opensource.org/osd) to determine whether or not a particular license meets their criteria here; in a nutshell, a license must meet the following criteria: 
+
+
+1. Free redistribution. 
+2. Availability of source code.
+3. Freedom to create derivative works 
+4. No discrimination against people or groups 
+5. No discrimination against use cases 
+
+
+Some of these clauses seem straightforward at first glance but in fact are not. For example, open source software can’t be restricted to use cases that the program author considers ‘good.’ An open source software can be used as part of a spyware or cyberwarfare stack for an enemy state; it can be used in weapons technology. Open source software can be used by the software author’s commercial competitors. It’s possible to limit who contributes to the software because contributions require approval from a maintainer, but it is not possible to limit who uses and benefits from the software. 
```

Comment:
To me this sounds a bit like a repetition of what was said in the sentence above (currently line 18). What do you think about combining the two. Personally, I like the sentences in line 34 a bit more.